README.md

```diff
@@ -1,12 +1,26 @@
-# Git Repository Template
+# KMM Templates
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+Kotlin Multiplatform project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
 
-## Usage
+## Prerequisite
 
-Clone the repository
+- Java 17
```

Comment:
How about mentioning all tools from this: https://kotlinlang.org/docs/multiplatform-mobile-setup.html#install-the-necessary-tools

Reply:
Fixed in https://github.com/nimblehq/kmm-templates/pull/11/commits/5ad4a3ba6ceb824980cf27c7e013587df8caf097